src/System.Security.Cryptography.Xml/ref/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
-      uap;
+      netstandard;
+      uap
```

Comment:
@ericstj correct me if I'm wrong but we shouldn't need a uap ref/src for this netstandard2.0 based library correct? It is possible that we are getting a nuget validation issue because uap doesn't yet map to netstandard2.0.  In which case I think I would prefer we temporarily remove uap10.1 from the list of supported frameworks in the pkgproj.